### PR TITLE
Add permissions for CSI driver

### PR DESCRIPTION
**Pull request by vcerenu**
This PR add instructions for upgrade EKS cluster to Kubernetes 1.23 or higher.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,15 @@ Deploy a Wazuh cluster with a basic indexer and dashboard stack on Kubernetes.
 
 ## Documentation
 
-The [instructions.md](instructions.md) file describes how to deploy Wazuh on Kubernetes.
+## Amazon EKS development
+
+To deploy a cluster on Amazon EKS cluster read the instructions on [local-environment.md](local-environment.md).
+Note: For Kubernetes version 1.23 or higher, the assignment of an IAM Role is necessary for the csi driver to function correctly. Within the AWS documentation you can find the instructions for the assignment: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html
```
Comment (teddytpc1) [inline README.md:21]: CSI should be in capital letters.

File: instructions.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ This guide describes the necessary steps to deploy Wazuh on Kubernetes.
     - Create Persistent Volumes on top of AWS EBS when using a volumeClaimTemplates
     - Create a record set in AWS Route 53 from a Kubernetes LoadBalancer.
 - Having at least two Kubernetes nodes in order to meet the *podAntiAffinity* policy.
+- For Kubernetes version 1.23 or higher, the assignment of an IAM Role is necessary for the csi driver to function correctly. Within the AWS documentation you can find the instructions for the assignment: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html
```
Comment (teddytpc1) [inline instructions.md:12]: CSI should be in capital letters.

File: README.md (hunk 1 above)
Comment (vcerenu) [reply README.md]: done

File: instructions.md (hunk 2 above)
Comment (vcerenu) [reply instructions.md]: done

**Review by teddytpc1 — APPROVED** [review]

**Review by c-bordon — APPROVED** [review]
LGTM

**Pull request merged**